src/main/views/govukTemplate.njk

```diff
@@ -4,7 +4,11 @@
 <!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
 <head>
   <meta charset="utf-8" />
-  <title>{{ t(title) }}</title>
+  {% if heading %}
+    <title>{{ t(heading) }} - {{ t(moneyClaims) }}</title>
+  {% else %}
+    <title>{{ t(moneyClaims) }}</title>
```

Comment:
why would you not have a heading here? moneyclaims probably isn't a great fallback

Reply:
It seems that when trying to set the heading on any confirmation page, the heading does not seem to render within the 'govuk-box-highlight' css

Reply:
this page should have the correct title though this is a hack which breaks it, i would suggest a special variable instead, confirmation_heading maybe?

Reply:
Sounds good